### ISSUE TITLE
Street Address Single Line Only in V 8.4.1

### ISSUE THREAD

**Issue by AndrewC3W** [salesagility/SuiteCRM-Core#384]
In SuiteCRM V7.x Street Address is a multi-line field were you can enter multiple street lines.
However in SuiteCRM V8.4.1 the Street Address is only a single line field.

SuiteCRM V8.x should have the same multi line Street Address field as in V7.x which was great.

See your own online V7.x and V8.x demo sites where the difference is glaringly obvious for targets, leads, contacts and accounts. Our local copy V8.4.1 is the exactly the same. 

There are very few UK addresses that only have a single Street Address which makes V8.x practically unusable here in the UK and I'm sure else where too.

Why did the developers decide not to implement multi line Street Address in V8.x ?  V8.x has been available for over a year and I would have expected it to be available by now as it is a core critical feature to make it usable.

Thanks

#### Your Environment

* SuiteCRM Version used: V8.4.1
* Browser name and version 109.0.5414.120 (Official Build) (64-bit):
* Environment name and version (e.g. MySQL 10.4.31-MariaDB, PHP 8.1
* Operating System and version Rocky Linux 8.8

**Comment by shubham-pawar** [comment]
Did you try it?

https://community.suitecrm.com/t/extra-street-address-fields/90277/11

**Comment by AndrewC3W** [comment]
Hello @shubham-pawar ,

I have tried the sample code as suggested in the post I started which you identified.

When you do a "quick repair and rebuild" that changes the datatype of the primary_address_street from varchar to text in the database. I'm assuming that varchar does not support the new line character ? Is that correct ?

In the Accounts module I can then enter a multi line primary_address_street.

And in Accounts I can also export a csv which does have a multi line primary_address_street. I have not attempted an import so far.

If the modification is revered then I assume primary_address _street will be changed back to a varchar and any mult iline primary_address_street  will be lost. Is that correct ?

Why is the primary_address_street datatype not a text by default ? Is this by design or is it a bug ?

Thanks.
